src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -276,6 +302,31 @@ export class CourseOverviewComponent implements OnInit, OnDestroy, AfterViewInit
         }
     }
 
+    /** initialize courses attribute by retreiving recently accessed courses from the server */
+    getRecentlyAccessedCourses() {
+        this.courseService.findAllForDashboard().subscribe({
```

Comment:
Please unsubscribe on Destroy

Reply:
done

Reply:
Where does the unsubscribe happen? I can't stop it in the code 😅

Reply:
in line 720 👍